From 18636794dcaab4a8a775046f093131c8361c8434 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Thu, 23 May 2013 15:45:40 +0200
Subject: [PATCH] spec: Deprecate spice-client

This has been true for quite some time, it's high time we stop
building spicec and have remote-viewer replace it on upgrades.
---
 virt-viewer.spec.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/virt-viewer.spec.in b/virt-viewer.spec.in
index cb0c9af..361511b 100644
--- a/virt-viewer.spec.in
+++ b/virt-viewer.spec.in
@@ -78,6 +78,11 @@ BuildRequires: intltool
 BuildRequires: libgovirt-devel >= 0.0.3
 %endif
 
+%if 0{?fedora} >= 20
+Obsoletes: spice-client < 0.12.3-2
+%endif
+
+
 %description
 Virtual Machine Viewer provides a graphical console client for connecting
 to virtual machines. It uses the GTK-VNC or SPICE-GTK widgets to provide
